src/Http/Middleware/Google2fa.php

```diff
@@ -65,4 +67,15 @@ public function handle($request, Closure $next)
 
         return response(view('google2fa::authenticate'));
     }
+
+    /**
+     * Set headers to NOT cache a page, used to prevent seeing a 2fa auth form
+     * when user clicks on the back button multiple times after logging out.
+     */
+    private function applyHeaders()
+    {
+        header("Expires: Thu, 19 Nov 1981 08:52:00 GMT"); //Date in the past
+        header("Cache-Control: no-store, no-cache, must-revalidate"); //HTTP/1.1
+        header("Pragma: no-cache"); //HTTP 1.0
```

Comment:
Apply the headers to the request, rather than outputting them directly here. The way it is here, makes it impossible to intercept/change that behaviour from the outside - e.g. by applying another middleware that will change one of those headers again to something else. This is a package, so needs to be flexible.

Reply:
Done! I couldn't add return types and arguments because of mixed types.

Reply:
No problem